src/main/java/uk/gov/hmcts/divorce/solicitor/service/SolicitorCreateApplicationService.java

```diff
@@ -109,4 +111,29 @@ public CaseData aboutToSubmit(
             .data(caseData)
             .build();
     }
+
+    public AboutToStartOrSubmitResponse<CaseData, State> setApplicant2SolOrganisationInfo(
```

Comment:
In my opinion you should just move this method to the callback handler

Reply:
will move it as a separate PR